.env

```
@@ -1 +1 @@
-VITE_GROQ_API_KEY=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
+VITE_GROQ_API_KEY=YOUR_API_KEY_HERE  # Replace with your actual key
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)